_commands/editors/vi.md

```diff
@@ -0,0 +1,39 @@
+vi
+
+
+
+'vi', also known as "vim" (Vi IMproved), is a text editor for programmers, designed to facilitate optimal efficiency when writing code.
+
+To open vi, simply type the following:
+
+~~~ bash
+$ vi [filename]
+~~~
+
+This will open a file with the name specified. If there is an existing file with that name, it will open that file, however if no file is found with that name, a new, empty one will be created.
+
+### Here are some useful things you can do in the editor:
+
+#### 'esc' - enter command mode
```

Comment:
I haven't rendered this on my end yet, but it looks like it will be a number of very large headers. It might be better with a bullet point list of keyboard shortcuts, like

  * <kbd>h</kbd>, <kbd>j</kbd>, <kbd>k</kbd>, <kbd>l</kbd> -- move cursor
  * <kbd>i</kbd> -- enter insert mode